### ISSUE TITLE
Nfc It appears as if NFC has been added but I can not find an example?

### ISSUE THREAD

**Issue by smittyb42** [eymengunay/php-passbook#96]
Nfc It appears as if NFC has been added but I can not find an example?

Are there examples on how to add Nfc to a pass?

**Comment by razvanphp** [comment]
Good point, we will add more info about this in the README, but here is the [Apple Developer documentation link](https://developer.apple.com/documentation/walletpasses/pass/nfc).

Basically it's just one extra json filed in the pass of type `Pass.NFC` with 3 properties: `encryptionPublicKey`, `message` and `requiresAuthentication`.

Please note the prerequisites for using this feature:

1. An Apple Developer account that has been approved to issue NFC Pass Type Identifier Certificates - you can request NFC activation from the [Apple Developer Portal](https://developer.apple.com/contact/passkit/) (login required).
2. An Apple NFC Pass Type Identifier Certificate.
3. A terminal or other hardware capable of reading Apple VAS and Google SmartTap Passes, together with compatible Point of Sale or other software to process the pass payload.

When requesting NFC activation, Apple will ask for detailed information about your intended use case and customer experience.  The following use cases are considered:

- Loyalty Cards
- Membership Cards
- Coupons
- Event Tickets
- Boarding Passes

Unattended access control or stored value cards are not considered.

Apple will also request evidence that you have the necessary hardware, software and capabilities to develop or deploy an ecosystem that is going to deliver an experience consistent with their guidelines.  You should provide as much detail as possible in your request and we recommend that you follow up with your regional Apple Worldwide Developer Relations Manager a few days after submission.
